apps/apps.c

```diff
@@ -2432,10 +2432,18 @@ BIO *dup_bio_out(int format)
 {
     BIO *b = BIO_new_fp(stdout,
                         BIO_NOCLOSE | (istext(format) ? BIO_FP_TEXT : 0));
+    void *prefix = NULL;
+
 #ifdef OPENSSL_SYS_VMS
     if (istext(format))
         b = BIO_push(BIO_new(BIO_f_linebuffer()), b);
 #endif
+
+    if ((prefix = getenv("HARNESS_OSSL_PREFIX")) != NULL) {
+        b = BIO_push(BIO_new(apps_bf_prefix()), b);
+        BIO_ctrl(b, BIO_C_SET_PREFIX, 0, prefix);
```

Comment:
I'm not sure repurposing `BIO_C_SET_PREFIX` for a completely different usage is wise

Reply:
Why?  Do you see us using this filter on top of the ASN.1 BIO?  That's really the only case I can see, and I have very strong doubts that will ever happen.

But sure,  I can add an apps local control, that wouldn't be hard.  Just gotta take a high enough number that we don't use yet and will most likely not use any time soon, right?

Reply:
Reusing a ctrl for a different purpose goes against the grain for me at a conceptual level. In reality its unlikely to cause any real problems in this case - but I just think its a practice we should avoid.

Yeah - just using a high enough number should do it.

Reply:
Done